tests/tests_modifications_with_selinux_disabled.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: Ensure the default is targeted, enforcing, without local modifications
+  hosts: all
+  gather_facts: true
+  vars:
+    selinux_all_purge: true
+    selinux_logins_purge: true
+
+  tasks:
+    - name: Ensure selinux-policy-targeted
+      package:
+        name:
+          - selinux-policy-targeted
+        state: present
+
+    - name: Add a Linux System Roles SELinux User
+      user:
+        comment: Linux System Roles SELinux User
+        name: sar-user
+    - name: Switch to permissive to allow login when selinuxfs is not mounted
+      command: setenforce 0
+      changed_when: true
+      when: ansible_selinux.status != "disabled" and
+        ansible_selinux.mode != "permissive"
+      register: selinux_switch_to_enforcing
+    - name: Get selinuxfs mountpoint
+      command: findmnt -n -t selinuxfs --output=target
+      changed_when: false
+      register: selinux_mountpoint
+    - name: >-
+        Umount selinux mountpoint to emulate SELinux disabled
+        system {{ selinux_mountpoint.stdout }}
+      command: umount -l {{ selinux_mountpoint.stdout }}
+      changed_when: true
+      when: selinux_mountpoint.stdout != ""
+
+    - name: Execute the role and catch errors
```

Comment:
The test isn't catching errors, it is "Execute the role and cleanup" - not sure if you are expecting any errors, but the test is failing:
```
TASK [linux-system-roles.selinux : Set SELinux file contexts] ******************
task path: /WORKDIR/git-test-modifications-in-disabled37f0mfqq/tests/roles/linux-system-roles.selinux/tasks/main.yml:74
Friday 06 October 2023  21:18:41 +0000 (0:00:00.013)       0:00:12.398 ******** 
failed: [sut] (item={'target': '/tmp/test_dir(/.*)?', 'setype': 'user_home_dir_t', 'ftype': 'd'}) => {
    "ansible_loop_var": "item",
    "changed": false,
    "item": {
        "ftype": "d",
        "setype": "user_home_dir_t",
        "target": "/tmp/test_dir(/.*)?"
    }
}

MSG:

SELinux is disabled on this host.
```

Reply:
That's expected because https://github.com/linux-system-roles/selinux/pull/194 is not merged yet. The test demonstrates the current problem and should start passing when https://github.com/linux-system-roles/selinux/pull/194 is merged

Reply:
`- name: Execute the role and catch errors` should be just `- name: Execute the role`